components/prism-dataweave.js

```diff
@@ -0,0 +1,45 @@
+(function (Prism) {    
+    Prism.languages.dataweave = {	
+        'url': {
+            pattern:/\b[A-Za-z]+:\/\/[\w/:.?=&-]+|\burn:[\w:.?=&-]+/
+        },
+        'property': {
+            pattern: /(?:\w+#)?(?:"(?:\\.|[^\\"\r\n])*"|\w+)(?=\s*[:@])/,
+            greedy: true
+        },
+        'string': {
+            pattern: /(["'`])(?:\\[\s\S]|(?!\1)[^\\])*\1/,
+            greedy: true
+        },
+        'mime-type': {
+            pattern: /\b(?:text|audio|video|application|multipart|image)\/[\w+-]+/
+        },       
```

Comment:
Just the regex is enough.

```suggestion
        'mime-type': /\b(?:text|audio|video|application|multipart|image)\/[\w+-]+/,
```